recipe/meta.yaml

```diff
@@ -101,6 +110,11 @@ outputs:
         # host C compilers work fine (better) on mac
         - {{ compiler('c') }}
     test:
+      # Ensure the correct dependencies are available for testing
+      requires:      
+        - mpi 1.0 openmpi
```

Comment:
This should not be needed, if so it's a bug in conda-build not in this recipe, we never had to add it. Let's keep it clean:
```suggestion
```